config/locales/views/submissions/en.yml

```diff
@@ -9,6 +9,7 @@ en:
       Would you like to continue?
     collect:
       apply_late_penalty: Apply late penalty/Deduct grace credits (Leaving this box unchecked means the submission will not be penalized, regardless of submission time.)
+      collect_current: 'Collect most recent submissions? <strong>Warning: This will collect the most recent submission, regardless of due date.</strong>'
```

Comment:
The convention is to add the `_html` suffix to keys that contain html. See the `override_existing_html` key below for an example. This should be renamed to `collect_current_html` (see here for more info: https://guides.rubyonrails.org/i18n.html#using-safe-html-translations)